skrf/network.py

```diff
@@ -4193,7 +4194,7 @@ def step_response(
             )
 
         t, y = self.impulse_response(window=window, n=n, pad=pad, bandpass=False, squeeze=squeeze)
-        return t, np.cumsum(y, axis=0)
+        return t, cumulative_trapezoid(y, initial=0, axis=0)
```

Comment:
Interesting. Does it gives a better agreement?

Reply:
So in fact I asked the guy behind Metas VNA Tools how they calculate the step response, now we have exact agreement.
In general using trapezoid integration is most of the time not worse than simple rectangle integration.

Reply:
Good point about the window, I checked manually that boxcar and None behave the same, but a test case would be fine as well